app/views/modals/set-home-page-modal.html

```diff
@@ -0,0 +1,50 @@
+<div class="modal-resource-action">
+  <div class="modal-header">
+    <h1>Set Home Page</h1>
+  </div>
+  <div class="modal-body modal-body-indent">
+    <p>
+      <div class="title">Set your preferred home page.</div>
+      Note: This setting is browser specific and will not be maintained across browsers.
+    </p>
+    <ng-form name="homePageForm">
```

Comment:
`<form>` instead of `<ng-form>`. `ng-form` should really only be used if it's nested inside another `form` element.